src/server.js

```diff
@@ -280,6 +280,20 @@ function init(socket, client) {
 				client.names(data);
 			}
 		);
+
+		socket.on("msg:preview:toggle", function(data) {
+			const networkAndChan = client.find(data.target);
+			if (networkAndChan) {
```

Comment:
Done. I honestly prefer the nested version, shorter and makes more sense to me, but whatever.

Adding `do not merge` label so I remember to squash before merging.